cpp/src/arrow/compute/kernels/scalar_validity.cc

```diff
@@ -126,12 +148,16 @@ void IsNullExec(KernelContext* ctx, const ExecBatch& batch, Datum* out) {
 
 const FunctionDoc is_valid_doc(
     "Return true if non-null",
-    ("For each input value, emit true iff the value is valid (non-null)."), {"values"});
+    ("For each input value, emit true if the value is valid (non-null)."), {"values"});
```

Comment:
"iff" means "if and only if", not typo?

Reply:
You are right! First time I see this notation. Will fix it (including following comments) tomorrow.

Reply:
Fixed in last commit.